src/App.css

```diff
@@ -41,24 +42,65 @@
 }
 
 .concentration-container {
-  font-size: 18px
+  font-size: 18px;
 }
 
 .title {
   font-size: 24px;
   margin-top: 10px;
 }
 
+#game_container {
```

Comment:
Minor nit - generally for CSS whitespaces can be represented as a dash - rather than an underscore _ so #game_container -> #game-container (note there isn't a real engineering benefit aside from making the naming conventions for your variables distinct from your classNames in css)